pta-intelligent-search-api/src/main/java/fi/maanmittauslaitos/pta/search/api/search/OntologyElasticsearchQueryProviderImpl.java

```diff
@@ -248,6 +253,28 @@ private QueryBuilder freetextQuery(String field, String word, double weight, dou
 		return query;
 	}
 
+    private void addSynonymQueries(Collection<String> searchTerms, BoolQueryBuilder boolQuery) {
+        for (String searchTerm : createSynonymList(searchTerms)) {
+            addFreeTextQuery(searchTerm, boolQuery);
+        }
+    }
+
+    private List<String> createSynonymList(Collection<String> searchTerms) {
+        List<String> synonymList = new ArrayList<>();
+        for(String term : searchTerms) {
+            List<String> termSynonyms = synonymsWords.get(term);
```

Comment:
style issue: getters are preferred over direct field access - helps reader keep track of object-level fields

Reply:
Added a getter for synonymWords field